.github/workflows/release.yml

```diff
@@ -118,6 +117,13 @@ jobs:
           release_name: Release ${{ env.PACKAGE_VERSION }}
           draft: false
           prerelease: false
+      - name: Dependency Graph
```

Comment:
Moved the order of this up so it wouldn't block the rest of the build after pypi is pushed.